org.eclipse.m2e.core.ui.tests/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Fragment-Host: org.eclipse.m2e.core.ui;bundle-version="[1.16.0,2.0.0)"
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: org.eclipse.m2e.core.ui.tests
-Bundle-Version: 1.18.1.qualifier
+Bundle-Version: 1.18.2.qualifier
```

Comment:
Is this necessary? 1.18.1 was not released yet, so we can most likely stick to 1.18.1

Reply:
Fixed, a wrong baseline was used.